File: qt/UTF-8/src/MainWindow.cpp
#include "MainWindow.h"
#include "ui_MainWindow.h"

////////////////////////////////////////////////////////////////////////////////

MainWindow::MainWindow( QWidget *parent ) :
    QMainWindow ( parent ),
    _ui ( new Ui::MainWindow() )
{
    _ui->setupUi( this );

    QString text = u8"Za\305\274\303\263\305\202\304\207 g\304\231\305\233l\304\205 ja\305\272\305\204";

    _ui->label->setText( text );
}

////////////////////////////////////////////////////////////////////////////////

MainWindow::~MainWindow()
{
    if ( _ui ) delete _ui;
    _ui = nullptr;
}

File: python/PySide/03_sample_project/sample_project/mainwindow.py
from PySide2.QtWidgets import QMainWindow
from PySide2.QtCore import Slot
from .ui_mainwindow import Ui_MainWindow


class MainWindow(QMainWindow):
    def __init__(self):
        super(MainWindow, self).__init__()
        self.ui = Ui_MainWindow()
        self.ui.setupUi(self)

    @Slot()
    def on_actionExit_triggered(self):
        self.close()

    @Slot()
    def on_pushButton_clicked(self):
        self.ui.label.setText("Push button clicked")
